btsieve/src/load_settings.rs

```diff
@@ -12,34 +13,30 @@ pub struct Opt {
 
 pub fn load_settings(opt: Opt) -> Result<Settings, ConfigError> {
     match opt.config_file {
-        Some(config_file) => {
-            if config_file.exists() {
-                Settings::read(config_file)
-            } else {
-                Err(ConfigError::Message(format!(
-                    "Could not load config file: {:?}",
-                    config_file
-                )))
+        Some(file) => parse_config_file(file),
+        None => match ProjectDirs::from("tech", "CoBloX", "btsieve") {
```

Comment:
Do we want this to say comit-network instead?
```suggestion
        None => match ProjectDirs::from("network", "comit, "btsieve") {
```

Reply:
Cool, I like it.

Reply:
Please also update the comments below.

Reply:
Without `network` (or `tech`) the path resolves to: 
`"/Users/<username>/Library/Preferences/comit-network.btsieve/btsieve.toml"
`

Reply:
> Please also update the comments below.

I don't understand what you mean @bonomat, can you explain please (if action is still required), thanks.